packages/api/package-lock.json

```diff
@@ -61,6 +65,17 @@
         "node": ">=14"
       }
     },
+    ".api/apis/petstore": {
+      "name": "@api/petstore",
+      "dependencies": {
+        "api": "../../../",
+        "oas": "^18.3.3"
+      }
+    },
+    ".api/apis/petstore/node_modules/api": {
```

Comment:
I re-ran `npm i` in this directory after modifying the `package.json` in the `petstore` subdirectory and it looks like the link worked?